openedx/features/edly/utils.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib.sites.shortcuts import get_current_site
+from openedx.features.edly.models import EdlySubOrganization
```

Comment:
Imports order: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#imports-order
```python
from django.contrib.sites.shortcuts import get_current_site

from openedx.features.edly.models import EdlySubOrganization
```

Reply:
File is updated. Still showing the old changes here.